src/main/scala/encry/network/NodeViewSynchronizer.scala

```diff
@@ -67,7 +66,7 @@ class NodeViewSynchronizer(influxRef: Option[ActorRef],
       }
     case AuxHistoryChanged(history) => historyReaderOpt = Some(history)
     case ChangedHistory(reader: EncryHistory@unchecked) if reader.isInstanceOf[EncryHistory] =>
-      deliveryManager ! ChangedHistory(reader)
+      deliveryManager ! HistoryChanges(reader)
```

Comment:
in this case you are sending new instance of history, not only changes. So, looks like name "ChangedHistory" is better